src/mote/templates/mainpage.html

```diff
@@ -72,12 +79,12 @@
             </div>
         </nav>
         <div class="container-lg">
-            <div class="card text-center" style="margin-top: 10%;">
+            <div class="card text-center" style="margin-top: 0%;">
```

Comment:
I do remember mentioning the use of prebuilt Bootstrap margin classes over hardcoded CSS margin statements here https://github.com/fedora-infra/mote/pull/232#pullrequestreview-839715765. Please follow the same and reduce the margin to the `mt-1` scale.